functions/Update-DbaInstance.ps1

```diff
@@ -189,7 +202,7 @@ function Update-DbaInstance {
             }
         }
         $actions = @()
-        $actionTemplate = @{}
+        $actionTemplate = @{ }
```

Comment:
please remove this space

Reply:
this space was added by vs code if i had to guess

Reply:
yeah my vs code likes to "format" powershell, I'll remove it.